lib/shopify_api/auth/oauth.rb

```diff
@@ -16,9 +16,10 @@ class << self
             shop: String,
             redirect_path: String,
             is_online: T.nilable(T::Boolean),
+            scope: AuthScopes,
```

Comment:
This should really be called `scopes` but we're already using `scope` to refer to the collection in multiple places, so I've followed that convention.